ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__ota-multi-stable-4.15-upgrade-from-stable-4.14.yaml

```diff
@@ -78,6 +78,21 @@ tests:
     - ref: cucushift-upgrade-healthcheck
     - ref: cucushift-ota-postupgrade
     workflow: cucushift-installer-rehearse-azure-ipi
+- as: aws-ipi-efa-pg-f28
+  cron: 32 15 5 * *
+  steps:
+    cluster_profile: aws-1-qe
```

Comment:
```suggestion
    cluster_profile: aws-qe
```

Reply:
What's the difference between aws-1-qe and aws-qe, I see all e2e jobs are using aws-1-qe.

Reply:
If I remember right, the aws-1-qe is for automated-release project, automated-release project selected a part of jobs to run, they have to be completed in some hours, to avoid slice quota limit issue, we created a dedicated cluster profile for it.

Reply:
Oh, sorry, maybe I was wrong, I saw most of ci jobs using `aws-1-qe` is for edge node testing. But I do not know why `aws-ipi-efa-pg` also use it, maybe some mis-configuration? cc @yunjiang29 to confirm.

Reply:
changed to aws-ipi, so we don't care about the cluster_profile anymore.

Reply:
> But I do not know why aws-ipi-efa-pg also use it,

there are some restrictions in placement group, see https://github.com/openshift/release/pull/41990#discussion_r1289943299

Reply:
> automated-release project 

`aws-autorelease-qe`